Dockerfile.compute-node

```diff
@@ -754,16 +773,27 @@ RUN rm /usr/local/pgsql/lib/lib*.a
 # Extenstion only
 #
 #########################################################################################
+FROM python:3.11-slim-bookworm AS generate-ext-index
+ARG PG_VERSION
+ARG BUILD_TAG
+# Define extension build numbers
+# NOTE: it is *not* necessary for the build tag to be BUILD_TAG. In particular,
+# you should update the build_tag for each extension only when you build it
+RUN echo "kq_imcx ${BUILD_TAG}" >>  build_numbers.txt
+RUN echo "anon ${BUILD_TAG}" >>  build_numbers.txt
+# RUN echo "postgis-build 5615610098" >>  build_numbers.txt
+# copy the control files here
+COPY --from=kq-imcx-pg-build /extensions/ /extensions/
+COPY --from=pg-anon-pg-build /extensions/ /extensions/
+# COPY --from=postgis-build /extensions/ /extensions/
```

Comment:
We need either revert postgis changes or add them to the image.
I think it's a good example of an extension with multiple control files inside a single archive.

Reply:
ok, I'll upload it.